blocksync/blocksync_test.go

```diff
@@ -40,7 +39,7 @@ import (
 	"github.com/iotexproject/iotex-core/testutil"
 )
 
-func newBlockSyncer(cfg config.BlockSync, chain blockchain.Blockchain, dao blockdao.BlockDAO, cs consensus.Consensus) (*blockSyncer, error) {
+func newBlockSyncer(cfg Config, chain blockchain.Blockchain, dao blockdao.BlockDAO, cs *mock_consensus.MockConsensus) (*blockSyncer, error) {
```

Comment:
why `MockConsensus`?

Reply:
blocksync is imported in the consensus package, use `consensus.Consensus` will cause an import cycle error.